Backpack/src/main/java/net/skyscanner/backpack/util/extensions.kt

```diff
@@ -0,0 +1,17 @@
+package net.skyscanner.backpack.util
+
+import android.animation.AnimatorInflater
+import android.os.Build
+import android.view.View
+import androidx.annotation.DrawableRes
+
+fun View.isStateListAnimatorSupported() =
```

Comment:
I don't think we need to make it public.

Reply:
I agree, internal is better